src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html

```diff
@@ -46,9 +46,9 @@ <h2 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
                     <div class="form-group-narrow flex-grow-1 ml-3">
                         <jhi-date-time-picker
                             labelName="{{ 'artemisApp.exercise.dueDate' | artemisTranslate }}"
-                            [(ngModel)]="fileUploadExercise.dueDate"
+                            [(ngModel)]="fileUploadExercise.dueDate!"
```

Comment:
Why is this necessary here but not in line 40 of the same file which does basically the same? (Also, the date can be undefined, right?)

Reply:
you are right, in this place it was not necessary